src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -162,6 +164,18 @@ private boolean setUpLogging(ContestConfig config) {
   // returns: set of winners from tabulation
   private Set<String> runTabulationForConfig(ContestConfig config) {
     Logger.log(Level.INFO, "Beginning tabulation for config: %s", configPath);
+    try {
+      BufferedReader reader = new BufferedReader(new FileReader(configPath));
+      String line = reader.readLine();
+      while(line != null) {
```

Comment:
Should we have log lines before and after, e.g. "Begin config file FILENAME" and "End config file"?

Reply:
Agree with this. I don't think the filename is necessary though, since it's output immediately before.
`Begin config file contents:`
`End config file contents.`